WordPress/Classes/Services/NotificationSyncMediator.swift

```diff
@@ -309,6 +319,25 @@ final class NotificationSyncMediator: NotificationSyncMediatorProtocol {
             }, completion: done, on: .main)
         })
     }
+
+    func toggleLikeForPostNotification(like: Bool,
```

Comment:
I think `isLike` will be misleading, because this doesn't convey whether the notification is liked or not, this conveys wether the user wants to like or unlike.

Reply:
We can use `isLike` though

Reply:
Updated in 5f06283e56f9ee620ba14a5074816ed04ff47491